timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResults.vue

```diff
@@ -236,32 +238,90 @@ export default {
 
         if (!perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name]) {
           // this timeline is not yet in the results for this analyzer: add it
-          perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name] = {
-            id: analysis.timeline.id,
-            name: analysis.timeline.name,
-            color: analysis.timeline.color,
-            verdict: analysis.result,
-            created_at: analysis.created_at,
-            last_analysissession_id: analysis.analysissession_id,
-            analysis_status: analysis.status[0].status,
+          if (perAnalyzer[analysis.analyzer_name].analyzerInfo.is_multi) {
+            // this analyzer is a multi analyzer, so add all results for this analysis session
+            multiResults = this.getAnalyzerMultiResults(analyses, analysis.analyzer_name, analysis.analysissession_id, analysis.timeline.id)
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name] = {
```

Comment:
You could shorten this a bit. Then it is easier to see which parts are unique to multi analyzers:

```javascript
const analyzerInfo = {
  id: analysis.timeline.id,
  name: analysis.timeline.name,
  color: analysis.timeline.color,
  last_analysissession_id: analysis.analysissession_id,
};

if (perAnalyzer[analysis.analyzer_name].analyzerInfo.is_multi) {
  // this analyzer is a multi analyzer, so add all results for this analysis session
  multiResults = this.getAnalyzerMultiResults(analyses, analysis.analyzer_name, analysis.analysissession_id, analysis.timeline.id)
  perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name] = {
    ...analyzerInfo,
    analysis_status: multiResults.analysis_status,
    results: multiResults.results,
  }
} else {
  // this is not a multi analyzer, so add the results for this timeline
  perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name] = {
    ...analyzerInfo,
    analysis_status: analysis.status[0].status,
    results: [
      {
        verdict: analysis.result,
        created_at: analysis.created_at,
        analysis_status: analysis.status[0].status,
       },
     ]
  }
}
            

```